packages/n4s/src/enforce/compounds/oneOf.js

```diff
@@ -0,0 +1,9 @@
+import runLazyRules from 'runLazyRules';
+
+/**
+ * @param {*} value   Value to be test against rules
+ * @param {Function[]} rules    Rules to validate the value with
+ */
+export default function oneOf(value, ...rules) {
+  return (rules) && (rules.filter(ruleGroup => runLazyRules(ruleGroup, value)).length === 1);
```

Comment:
The first condition doesn't check anything here. Since `rules` is an arguments spread, it will always be an array.
```suggestion
  return rules.filter(ruleGroup => runLazyRules(ruleGroup, value)).length === 1;
```